packages/p2p/src/components/buy-sell/buy-sell-row.scss

```diff
@@ -90,6 +90,14 @@
 
     &__button {
         align-self: flex-start;
+        display: flex;
+        flex-flow: row wrap;
+        white-space: inherit;
```

Comment:
Added `white-space: inherit` because if the text overflows, the default css for button `dc-btn` has `white-space: nowrap` thus causing text to overflow to the right side:
<img width="275" alt="Screenshot 2022-07-18 at 2 28 34 PM" src="https://user-images.githubusercontent.com/103016120/179455534-f6523df0-7c2e-4da5-90e1-e634e5217f12.png">

### Before
<img width="453" alt="Screenshot 2022-07-18 at 2 29 17 PM" src="https://user-images.githubusercontent.com/103016120/179455638-1fe2d469-9a20-4bad-bc6a-346b6b4fe025.png">

### After
<img width="452" alt="Screenshot 2022-07-18 at 2 29 21 PM" src="https://user-images.githubusercontent.com/103016120/179455658-8eb46885-d97f-46ca-903b-42a36212b7ac.png">